SR2021/2020-10-29-sr2021-announcement.md

```diff
@@ -0,0 +1,26 @@
+---
+to: Potential team leaders
+subject: Announcing Student Robotics 2021
+---
+
+Hello there!
+
+I'm pleased to announce some details for Student Robotics 2021.
+
+## Competition
+
+Unfortunately, we are unable to run a physical competition in a safe way. Instead, we will be running a virtual competition in a simulated environment. The competition year will be very similar to what you're used to. In a few weeks, we'll kick off the competition with a kickstart, which will culminate in a final competition 1st May.
```

Comment:
Feel like we should highlight previous virtual competition. Reworded last sentences as well but take the changes or leave them I don't feel super strongly

```suggestion
Unfortunately, we are unable to find a way run a physical competition in a COVID safe way. Instead, following on the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/) we will be running the competition in a simulated environment. The competition's time scale will be very similar to what you're used to, with just a few short weeks until we'll kick off the competition with our Kickstart event. Teams will then have roughly 7 months before the final competition 1st May.
```